v2f/src/it/scala/org/broadinstitute/monster/etl/v2f/V2FIntegrationSpec.scala

```diff
@@ -0,0 +1,94 @@
+package org.broadinstitute.monster.etl.v2f
+
+import better.files._
+import com.spotify.scio.testing._
+import io.circe.Json
+import org.scalatest.BeforeAndAfterAll
+import org.apache.beam.sdk.options.PipelineOptionsFactory
+import org.scalatest.Matchers
+
+class V2FIntegrationSpec extends PipelineSpec with Matchers with BeforeAndAfterAll {
+
+  behavior of "V2F ETL Pipeline"
+
+  private val truthDir = "v2f" / "src" / "it" / "test-files" / "outputs"
+  private val compareDir = "v2f" / "src" / "it" / "test-files" / "outputs-to-compare"
+  private val inputDirString = "v2f/src/it/test-files/inputs/"
+  private val compareDirString = "v2f/src/it/test-files/outputs-to-compare"
```

Comment:
Why `"v2f" / "src" / ...` style in one place, and `"v2f/src/..."` style in another?

Reply:
`truthDir` and `compareDir` are type `File` whereas the other two are type `String`. As far as I can tell, I need the `String`s in a spot or two and I need the `File`s in a spot or two. If they can be consolidated to just one type, then sure! I think I need both unless I am mistaken.